lesson_12/homework_12.py

```diff
@@ -16,3 +16,80 @@
     декількох дій, вивести в кого найбільше енергії лишилося.
 Створити тести на клас та на атрибути класу.
 """
+eating_cost = -5
+sleeping_cost = -10
+talking_cost = 5
+walking_cost = 10
+doing_homework_cost = 90
```

Comment:
```suggestion
eating_cost = -5
sleeping_cost = -10
talking_cost = 5
walking_cost = 10
doing_homework_cost = 90
```
ці змінні повинні бути частиною классу